frontends/web/src/api/account.ts

```diff
@@ -440,6 +440,16 @@ export const connectKeystore = (code: AccountCode): Promise<{ success: boolean;
   return apiPost(`account/${code}/connect-keystore`);
 };
 
+export type TResponseKeystoreName = {
+  success: boolean;
+  keystoreName?: string;
+}
+
+export const keystoreName = (rootFingerprint: string): Promise<TResponseKeystoreName> => {
```

Comment:
since you are here, ideally always return Fail | Success, that way keystoreName does not have to be optional since we know it is always a string in case of success. Also see other response types.